_posts/2023-01-23-OpenSearch-Flink-Connector.markdown

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title:  "Apache Flink® Connector for OpenSearch®"
+authors:
+- ftisiot
+- reta
+date: 2022-01-31
+categories:
+ - technical-post
+
+excerpt: "Apache Flink® and OpenSearch® are widely known and successful open source projects. Even if you have never used them, it is likely you have heard or read about them in search or streaming data transformation use cases. And there are reasons for that: although they target different tech markets, they perform their job very well."
+---
+
```

Comment:
Please add front matter here using the provided below: 

**Meta Keywords:** Apache Flink Connector to OpenSearch, open source data processing, Aiven for Apache Flink
**Meta Description:**  Learn how Aiven uses the Apache Flink Connector data processing solution to write from Apache Flink to an index of the OpenSearch engine.